crates/solidity/outputs/cargo/crate/src/unit_tests.rs

```diff
@@ -1,6 +1,10 @@
 use semver::Version;
-use slang_solidity::kinds::TokenKind::*;
-use slang_solidity::language::{Language, LexicalContext};
+
+use crate::kinds::LexicalContextType;
+use crate::kinds::TokenKind::*;
+use crate::language::Language;
+use crate::lexer::Lexer;
+use crate::support::ParserContext;
 
 #[test]
 fn test_next_token() {
```

Comment:
TOL, If we already (or can possibly) test these through the public parser API, do we even need this unit test here (and exposing `recovery::RecoverFromNoMatch`)?